### PR TITLE
zh: add config command for v2.0.5

### PR DESCRIPTION
**Pull request by GMHDBJD**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-dm) that's required for repo owners to accept my contribution.

### Which DM version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v2.0 (TiDB DM 2.0 versions)
- [ ] v1.0 (TiDB DM 1.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: zh/dmctl-introduction.md [hunk 1]
```diff
@@ -91,6 +92,7 @@ export DM_MASTER_ADDR="172.16.30.14:8261"
 ```
 Available Commands:
   check-task            check-task <config-file> [--error count] [--warn count]
+  config                commands to import/export config
```
Comment (lance6716) [inline zh/dmctl-introduction.md:95]: this command contains subcommands so didn't display syntax?

**Review by lance6716 — APPROVED** [review]

File: zh/dmctl-introduction.md (hunk 1 above)
Comment (GMHDBJD) [reply zh/dmctl-introduction.md:95]: Should use `help config` or `config help`

File: zh/maintain-dm-using-tiup.md [hunk 2]
```diff
@@ -179,6 +179,13 @@ tiup dm scale-in prod-cluster -N 172.16.5.140:8262
 
 ## 滚动升级
 
+> **注意：**
+>
+> 从 v2.0.5 版本开始，dmctl 支持[导出和导入集群的数据源和任务配置](export-import-config.md)
+> 升级前，可使用 `config export` 命令导出集群的配置文件，升级后如需降级回退到旧版本，可重建旧集群后，使用 `config import` 导入之前的配置。
+> 对于 v2.0.5 之前版本的集群，可使用 v2.0.5 及之后版本的 dmctl 导出和导入集群配置。
+> 对于 v2.0.2 之后的版本，导入集群配置时暂不支持自动恢复 relay worker 相关配置，可手动执行 `start-relay` 命令[开启 relay log](relay-log.md#启动停止-relay-log)。
+
```
Comment (Liuxiaozhen12) [inline zh/maintain-dm-using-tiup.md:188]:
```suggestion
> **注意：**
>
> 从 v2.0.5 版本开始，dmctl 支持[导出和导入集群的数据源和任务配置](export-import-config.md)。
>
> 升级前，可使用 `config export` 命令导出集群的配置文件，升级后如需降级回退到旧版本，可重建旧集群后，使用 `config import` 导入之前的配置。
>
> 对于 v2.0.5 之前版本的集群，可使用 v2.0.5 及之后版本的 dmctl 导出和导入集群配置。
>
> 对于 v2.0.2 之后的版本，导入集群配置时暂不支持自动恢复 relay worker 相关配置，可手动执行 `start-relay` 命令[开启 relay log](relay-log.md#启动停止-relay-log)。

```
最后一行 2.0.2 是不是写错了？应该是 2.0.5 ？

Comment (GMHDBJD) [reply zh/maintain-dm-using-tiup.md]: 是 2.0.2

**Review by Liuxiaozhen12 — APPROVED** [review]

**Pull request merged**